dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -43,6 +43,10 @@ class Teacher1819Application < ApplicationBase
     include SerializedProperties
     include SchoolInfoDeduplicator
 
+    OTHER_WITH_TEXT = TEXT_FIELDS[:other_with_text]
+    OTHER_PLEASE_LIST = TEXT_FIELDS[:other_please_list]
+    OTHER_PLEASE_EXPLAIN = TEXT_FIELDS[:other_please_explain]
```

Comment:
is it necessary to redeclare these here (and in the principal approval application)?

Reply:
No, I just did that to make the change easier to make & review. I'll remove them.